homeassistant/components/transmission/config_flow.py

```diff
@@ -59,18 +59,14 @@ async def async_step_user(self, user_input=None):
                 if (
                     entry.data[CONF_HOST] == user_input[CONF_HOST]
                     and entry.data[CONF_PORT] == user_input[CONF_PORT]
-                ):
-                    return self.async_abort(reason="already_configured")
-                if entry.data[CONF_NAME] == user_input[CONF_NAME]:
-                    errors[CONF_NAME] = "name_exists"
-                    break
-
+                ) or (entry.data[CONF_NAME] == user_input[CONF_NAME]):
+                    return self.async_abort(reason="single_instance_allowed")
             try:
                 await get_api(self.hass, user_input)
 
             except AuthenticationError:
-                errors[CONF_USERNAME] = "wrong_credentials"
-                errors[CONF_PASSWORD] = "wrong_credentials"
+                errors[CONF_USERNAME] = "invalid_auth"
+                errors[CONF_PASSWORD] = "invalid_auth"
```

Comment:
Where is the invalid_auth in `strings.json` ?

Reply:
Sorry, had unsaved changes. Added in next revision.